src/components/Hero/Hero.module.scss

```diff
@@ -85,3 +85,49 @@
   height: 100%;
   background-color: rgba(0, 0, 0, 0.3);
 }
+
+@media (max-width: 600px) {
+  .heroFooter {
+    min-width: 100vw;
+    font-size: 20vw;
+    gap: 1vh;
+    right: 0;
+    flex-direction: column;
+    bottom: 14;
+    padding: 0 1vw 2vh 0;
+  }
+
+  .heroType {
+    width: 95vw;
+    height: 9vh;
+  }
+
+  .header {
+    position: fixed;
+    top: 0;
+  }
+
+  .imageWrapper {
+    overflow: hidden;
+    max-height: 100vh;
+    height: 100vh;
+    max-width: 100vw;
+  }
+
+  .logoH1,
```

Comment:
more descriptive naming: logoLabel, logoName for example

Also you probably want a separate component/css file for the logo since it'll be used in other spots other than the homepage probably? And even if we only have 1 page, it would be nice to encapsulate it into it own folder/files